versions.tf

```diff
@@ -5,6 +5,9 @@ terraform {
     azurerm = {
       source = "hashicorp/azurerm"
     }
+    azuread = {
+      source = "hashicorp/azuread"
+    }
```

Comment:
I'm curious, why it's needed now/for this PR?

Reply:
it is an error during my `git add -p` sequence: removing it (I'll do a subsequent PR for it)